### ISSUE TITLE
loss 

### ISSUE THREAD

**Issue by AlOo2002** [Sylv-Lej/frcnn#1]
at the training the loss result is negative? could be or i am doing something wrong

**Comment by Sylv-Lej** [comment]
some training code is missing, i'll add it soon :)
